static/js/subs.js

```
@@ -938,7 +948,7 @@ $(function () {
         var stream_name = $sub_row.find('.subscription_name').text();
         var description = $sub_row.find('input[name="description"]').val();
 
-        $('#subscription_status').hide();
+        $('#subscriptions-status').hide();
```

Hmm, I guess the previous code here for hiding errors was just buggy?